### DISCUSSION TITLE
Value Area? Extended? Volume Profile feature

### DISCUSSION THREAD

**Discussion by AirForceFive** [highfestiva/finplot#508] [General]
Hello,

i have a lot of ideas but one of them being VAH VAL where personally you've done a great job for POC, and if it's possible to add a line for VAH and VAL, and also being extendable  as shown in the screenshot, for example in your current volume profile it is set for 'W' so only at the end of the week it'll extend and stop when price touches those value. I was wondering if something like that could be added it would be great :) You have some awesome stuff here but i'm no coder.

<img width="1328" alt="Screenshot 2024-03-21 at 4 20 49 PM" src="https://github.com/highfestiva/finplot/assets/130403336/6a6e59a8-9231-4cde-ab6b-1b0da50e9507">

**Comment by highfestiva** [comment]
Sure, you could. In the `volume-profile.py` example you just set `draw_poc` to some large value. Like so:

```python3
fplt.horiz_time_volume(time_volume_profile, draw_va=0, draw_body=0.0, draw_poc=10.0)
```
![image](https://github.com/highfestiva/finplot/assets/160303/b940f465-b540-428d-ac3b-f44ce52b4dbe)
